README.md

```diff
@@ -209,7 +209,7 @@ spec:
     }
 ```
 
-Can be found at: `example/deploy/crds/mellanox.com_v1alpha1_macvlannetwork_cr.yaml`
+Can be found at: `./deploy/crds/mellanox.com_v1alpha1_macvlannetwork_cr.yaml`
```

Comment:
can you copy the file under example folder  (all example files are there)? i think it was just missing.

Reply:
It will lead to maintaining two copies of the file. Do we want it?

Reply:
i would change it to some generic placeholder under /deploy/  similarly to what was done in:

https://github.com/Mellanox/network-operator/blob/master/deploy/crds/mellanox.com_v1alpha1_nicclusterpolicy_cr.yam

Reply:
the operator-framework creates some "sample" CRs when auto-generating the CRDs, these usually should have no meaningful content. i must have missed that mamduh put this here and not under example folder

Reply:
It's already in /deploy directory: https://github.com/Mellanox/network-operator/tree/master/deploy/crds